script/DeployLeveragedEthMainnet.s.sol

```diff
@@ -1,73 +1,38 @@
 // SPDX-License-Identifier: AGPL-3.0
 pragma solidity ^0.8.13;
 
-import {CREATE3Script} from "./base/CREATE3Script.sol";
-import {ERC20} from "solmate/tokens/ERC20.sol";
-import {WETH} from "solmate/tokens/WETH.sol";
-import {IPool} from "aave-v3/interfaces/IPool.sol";
-import {IAToken} from "aave-v3/interfaces/IAToken.sol";
-import {IPoolDataProvider} from "aave-v3/interfaces/IPoolDataProvider.sol";
+import "forge-std/console2.sol";
 
-import {Constants as C} from "../src/lib/Constants.sol";
-import {ICurvePool} from "../src/interfaces/curve/ICurvePool.sol";
-import {ILido} from "../src/interfaces/lido/ILido.sol";
-import {IwstETH} from "../src/interfaces/lido/IwstETH.sol";
-import {AggregatorV3Interface} from "../src/interfaces/chainlink/AggregatorV3Interface.sol";
-import {IVault} from "../src/interfaces/balancer/IVault.sol";
-import {ISwapRouter} from "../src/interfaces/uniswap/ISwapRouter.sol";
+import {DeployLeveragedEth} from "./base/DeployLeveragedEth.sol";
 import {scWETH} from "../src/steth/scWETH.sol";
 import {scUSDC} from "../src/steth/scUSDC.sol";
 import {sc4626} from "../src/sc4626.sol";
-import {MainnetDeployBase} from "./base/MainnetDeployBase.sol";
+import {Constants as C} from "../src/lib/Constants.sol";
+import {ISwapRouter} from "../src/interfaces/uniswap/ISwapRouter.sol";
 
-contract DeployScript is MainnetDeployBase {
-    function run() external returns (scWETH scWeth, scUSDC scUsdc) {
-        vm.startBroadcast(deployerPrivateKey);
+contract DeployScript is DeployLeveragedEth {
```

Comment:
Change the contract name to match the file name unless you have a particular reason for doing so? Same for the other 2 scripts.